src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/InputCommentsIndentationWithInMethodCallWithSameIndent.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+/**
+ * Config: default
+ */
+public class InputCommentsIndentationWithInMethodCallWithSameIndent {
+    String s1 = "ONE", s2 = "TWO", s3 = "THREE";
+    private final boolean myList = equals(
+        // Some comment here
+        s1,
+        s2
+        // ok
```

Comment:
Please add some bad cases where comment is before correct position and after correct position.

Reply:
Done.